srtcore/epoll.cpp

```diff
@@ -328,8 +334,7 @@ int CEPoll::update_usock(const int eid, const SRTSOCKET& u, const int* events)
 
             // Update the watch configuration, including edge
             wait.watch = evts;
-            if (edgeTriggered)
-                wait.edge = evts;
```

Comment:
Is this change intended? No need to check for `edgeTriggered`?

Reply:
This is another bugfix. This version was mistakenly set either all to edge-triggered or all to level-triggered depending on found `SRT_EPOLL_ET` flag, which didn't work correctly for flags that were always edge-triggered. The `et_evts` as set earlier was set correctly: to represent either ALL events (if `SRT_EPOLL_ET`) was found, or to represent only the "always-edge" events (that is, `SRT_EPOLL_UPDATE`).